app/index.jsx

```diff
@@ -2,8 +2,23 @@ var React = require('react')
 var ReactDOM = require('react-dom')
 const constants = require('./scripts/constants.js')
 
-var Nav = require('./components/nav.jsx')
-var BootstrapPre = require('./components/bootstrap-pre.jsx')
+var Navigation = require('./components/navigation.jsx')
```

Comment:
Because we have Nav, Navbar components. All the components are a part of the navigation in the view. So, I thought it would be apt